jdk/test/javax/swing/system/6799345/TestShutdown.java

```diff
@@ -21,13 +21,16 @@
  * questions.
  */
 
-/* @test
-   @bug 6799345
-   @summary Tests that no exceptions are thrown from TimerQueue and
-SwingWorker on AppContext shutdown
-   @author art
-   @run main TestShutdown
-*/
+/**
+ * @test
+ * @bug 6799345
+ * @key headful
+ * @summary Tests that no exceptions are thrown from TimerQueue and
+ * SwingWorker on AppContext shutdown
+ * @author art
+ * @modules java.desktop/sun.awt
```

Comment:
It appears that this and other related parts are copied from the jdk10 patch. "@modules java.desktop/sun.awt"/etc was part of the header in jdk10 before JDK-8185500, so it is part of the header after the patch. But in jdk8 we don't have it, so we shouldn't add it.

Reply:
"@modules java.desktop/sun.awt" correlation has been deleted.